src/App.js

```diff
@@ -23,29 +23,31 @@ export default function App() {
   const classes = useStyles();
 
   const [shouldRunSUSHI, setShouldRunSUSHI] = useState(false);
-  const [text, setText] = useState('Edit FSH Here!');
+  const [inputText, setInputText] = useState('Edit FSH Here!');
+  const [outputText, setOutputText] = useState('Your JSON Output Will Display Here: ');
 
-  function updateShouldRunSUSHI(shouldRunSUSHI) {
+  function buttonFunctionality(shouldRunSUSHI, sushiOutput) {
     setShouldRunSUSHI(shouldRunSUSHI);
+    setOutputText(sushiOutput);
   }
-  function updateTextValue(text) {
-    setText(text);
+  function updateInputTextValue(text) {
+    setInputText(text);
   }
 
   return (
     <div className="root">
       <TopBar />
-      <RunButton onClick={updateShouldRunSUSHI} text={text} />
+      <RunButton onClick={buttonFunctionality} text={inputText} />
       <Grid className={classes.container} container>
         <Grid className={classes.itemTop} item xs={6}>
           <CodeMirrorComponent
-            value={text}
-            updateTextValue={updateTextValue}
-            updateShouldRunSUSHI={updateShouldRunSUSHI}
+            value={inputText}
+            updateTextValue={updateInputTextValue}
+            updateShouldRunSUSHI={buttonFunctionality}
           />
         </Grid>
         <Grid className={classes.itemTop} item xs={6}>
-          <JSONOutput shouldDisplaySUSHI={shouldRunSUSHI} text={text} />
+          <JSONOutput shouldDisplaySUSHI={shouldRunSUSHI} text={outputText} />
```

Comment:
This is a minor comment, and probably a matter of preference, so you can choose to ignore it, but... is "should" really needed in the variable names?  Isn't this working kind of like a command?  Would `displaySUSHI` (and `runSUSHI`) be just as good (but shorter)?

Reply:
A good point. I changed the shouldDisplaySUSHI to just displaySUSHI, however I think we should keep shouldRunSUSHI with the should in front of it as the name of our function to execute sushi is now runSUSHI(). We may just want to avoid confusion between the boolean value it represents and the actual calling of the function runSUSHI()?

Reply:
That makes sense... That said, I feel like "should" implies that there is a chance it _won't_ though (like, "it _should_ run SUSHI, but really, who knows?!").  Maybe `doRunSushi` would be better?  I don't know -- I'm probably just getting nit-picky with semantics at this point!